cmd/up.go

```diff
@@ -740,6 +747,52 @@ func mergeDevPodUpOptions(baseOptions *provider2.CLIOptions) error {
 	return nil
 }
 
+func mergeEnvFromFiles(baseOptions *provider2.CLIOptions) error {
+	var variables []string
+	for _, file := range baseOptions.WorkspaceEnvFile {
+		envFromFile, err := parseKeyValueFile(file)
+		if err != nil {
+			return err
+		}
+		variables = append(variables, envFromFile...)
+	}
+	baseOptions.WorkspaceEnv = append(baseOptions.WorkspaceEnv, variables...)
+
+	return nil
+}
+
+func parseKeyValueFile(filename string) ([]string, error) {
```

Comment:
First time contributing so I'm not very familiar with the codebase, let me know if there's a generic package to put this bit in

Reply:
Hi @eduardodbr you could put this parse function in `pkg/devcontainer/config/` package :+1:

Reply:
done, thanks